crowbar_framework/lib/crowbar/backup/restore.rb

```diff
@@ -66,10 +66,20 @@ def restore_chef
       def restore_crowbar
         Crowbar::Backup::Base.restore_files.each do |source, destination|
           # keep the permissions of the files that are already in place
+          src_path = @data.join("crowbar", source)
+          dest_path = Pathname.new(destination)
+          # If source and destination are both directories we just need to
+          # copy the contents of source, not the directory itself.
+          src_string = if dest_path.directory? && src_path.directory?
+              "#{srcpath}/."
```

Comment:
Style/IndentationWidth: Use 2 (not 4) spaces for indentation. (https://github.com/bbatsov/ruby-style-guide#spaces-indentation)